### ISSUE TITLE
Can't clone repo after downloading ca certificate

### ISSUE THREAD

**Issue by shotor** [Yannik/qnap-letsencrypt#9]
**Platform/Firmware Information**

```
$ grep -e ^Platform -e ^DISPLAY_NAME  /etc/platform.conf
Platform = X86_BRASWELL
DISPLAY_NAME=TS-253A

$ grep -e ^Version -e ^Build -e Model -e "\[" /etc/default_config/uLinux.conf | grep -v "\[System\]" | awk '1;/\[/{exit}' |grep -v "\["

Model = TS-X53II
Internal Model = TS-X53II
Version = 4.2.1
Build Number = 20160419
Rsync Model = QNAP
Build Date = 2016-04-19
```

**Is this the latest firmware for your device?**
Yes

**Issue Summary (provide relevant error messages and log output):**

Following the steps outlined in the readme:

- SHA1 checks out

I set git to use the provided cacert, but when I try to clone the repo I get:

```
 $ git clone https://github.com/Yannik/qnap-letsencrypt.git
Cloning into 'qnap-letsencrypt'...
fatal: unable to access 'https://github.com/Yannik/qnap-letsencrypt.git/': SSL certificate problem, verify that the CA cert is OK. Details:
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed
```

**Comment by Yannik** [comment]
What's the output of
```
git config http.sslCAinfo
git config --system http.sslCAinfo
ls $(git config http.sslCAinfo)
ls $(git config --system http.sslCAinfo)
```

**Comment by ozhan99** [comment]
[/share] # git config http.sslCAinfo
/share/Software/letsencrypt/cacert.pem
[/share] # git config --system http.sslCAinfo
/share/Software/letsencrypt/cacert.pem
[/share] # head $(git config http.sslCAinfo)
##
## Bundle of CA Root Certificates
##
## Certificate data from Mozilla as of: Wed Apr 20 03:12:05 2016
##
## This is a bundle of X.509 certificates of public Certificate Authorities
## (CA). These were automatically extracted from Mozilla's root certificates
## file (certdata.txt).  This file can be found in the mozilla source tree:
## http://hg.mozilla.org/releases/mozilla-release/raw-file/default/security/nss/lib/ckfw/builtins/certdata.txt
##
[/share] # head $(git config --system http.sslCAinfo)
##
## Bundle of CA Root Certificates
##
## Certificate data from Mozilla as of: Wed Apr 20 03:12:05 2016
##
## This is a bundle of X.509 certificates of public Certificate Authorities
## (CA). These were automatically extracted from Mozilla's root certificates
## file (certdata.txt).  This file can be found in the mozilla source tree:
## http://hg.mozilla.org/releases/mozilla-release/raw-file/default/security/nss/lib/ckfw/builtins/certdata.txt
##

(I'm the one with the issue, the other account was of a colleague of mine)

**Comment by Yannik** [comment]
@ozhan99 Hm, that certainly looks fine. Can you reproduce this issue every time? Maybe you made a typo/missed a command the first time.

**Comment by ozhan99** [comment]
@Yannik I just copy paste everything from the manual you wrote, but still the same fatal error message. Do I need to upgrade or install something?

**Comment by ozhan99** [comment]
@Yannik Do you know where I can find the config file of Git? I found one at /share/CE_CACHEDEV1_DATA/.qpkg/git/repository/etc/ named "gitconfig", but it looks like that is not the correct one. When I do for example "git config --system http.sslVerify false", the information in this file does not change. 

Something else, After installing Entware-NG, I also did
> # opkg install ca-certificates
To install ca-cartificates.

The problem has actually more effects, I cannot do any git action anymore to any https site. Therefore some other apps do not work anymore (eg. SickRage). I really hope you have any tips for me. 

ps. wget to https works fine.

**Comment by Yannik** [comment]
How did you install git?

**Comment by ozhan99** [comment]
I used the Git from apps.qnap.community. But I did some actions today: 
- I downgraded my QTS to 4.2.1 20160311
- I removed Git from app center and reinstalled (if you download the file, watch-out, the file you download is actually a html file. The actually .qpkg file is downloadable from [here](https://www.dropbox.com/s/fiay536scdscew7/git_2.1.0_x86.qpkg)

So I don't know if I upgrade the QTS, everything again collaps, but I don't want to try. Everything is working now fine.
